pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -38,6 +39,18 @@ func getDefaultInstrumentation(agentConfig *adapters.CwaConfig, isWindowsPod boo
 	if !ok {
 		return nil, errors.New("unable to determine dotnet instrumentation image")
 	}
+	autoInstrumentationConfigCpu, _ := os.LookupEnv("AUTO_INSTRUMENTATION_LIMIT_CPU")
+	autoInstrumentationConfigMemory, _ := os.LookupEnv("AUTO_INSTRUMENTATION_LIMIT_MEMORY")
+
+	var autoInstrumentationConfigLimits corev1.ResourceList
+	if autoInstrumentationConfigCpu == "" || autoInstrumentationConfigMemory == "" {
```

Comment:
Does this logic make sense? If CPU is set and Memory isn't, or vice versa, it will default to the OS's default values for both since that is what happens when it's set to `nil` as a whole. Is that okay?